### PR TITLE
Project migration to webpack 5

### LINKED ISSUE #10668: Upgrade environment to use webpack 5
## Provide a description of the task

Scope: https://github.com/ckeditor/ckeditor5/issues/9799 (Tools)

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Other: Project migration to webpack 5. Closes #10668.

---

### Additional information

This PR requires changes from https://github.com/ckeditor/ckeditor5-dev/pull/729.

Together with this PR, the following should be closed:
* Internal and CF repositories.
* [Package Generator](https://github.com/ckeditor/ckeditor5-package-generator/pull/57).
* Online Builder.
* Umberto.
* CS (an update in the testing env. for the client package).

### CODE REVIEW

**Comment by elliottregan** [comment]
It would be really nice to get these updates merged in...Is there a timeline for this?

**Comment by pomek** [comment]
We will go back to this topic next week. It means that the release in January will contain changes related to webpack 5.

**Comment by elliottregan** [comment]
Thanks for the update! Looking forward to it.

File: docs/builds/guides/integration/advanced-setup.md [hunk 1]
```diff
@@ -148,22 +149,29 @@ module.exports = {
...
+					}
 				]
 			}
 		]
 	}
 };
 ```
 
+<info-box>
+    If you cannot use the latest webpack (when writing this guide, it is 5), the provided configuration will also work with webpack 4.
```
Comment (godai78) [inline docs/builds/guides/integration/advanced-setup.md:172]:
```suggestion
    If you cannot use the latest webpack (at the moment of writing this guide, it is 5), the provided configuration will also work with webpack 4.
```

**Review by godai78 — APPROVED** [review]

**Pull request merged**